### DISCUSSION TITLE
Unreal Engine 4 Titles

### DISCUSSION THREAD

**Discussion by TheNewJavaman** [TheNewJavaman/unreal-vr#26] [General]
Thanks to Mister on Discord for creating this list of ideal VR UE4 games. Let's see when your GitHub account gets unbanned 😅 

[Google Sheets link](https://docs.google.com/spreadsheets/d/13_8Epp8LIRwY4CNQZCdcRzfjZaQqzus8UdJ1hDtQkpc/edit#gid=0)

**Comment by Robert-W-K** [comment]
I am unflagged now! H-M got posted today with notes.
